src/components/Modal/Modal.scss

```diff
@@ -3,7 +3,6 @@ $small-width: rem(620px);
 .BodyWrapper {
```

Comment:
Make sure to add an entry in the changelog.
For example https://github.com/Shopify/polaris-react/pull/1580/files#diff-828bfc9e3c18f7cacda75343f4a1d75e

Reply:
Do you think it warrants a changelog entry? For now I added the `skip changelog` label because I thought it does not.

Reply:
Not sure. Better to check with the Polaris team.

Reply:
Since we're unsure why this was here, I think it warrants a changelog entry. That way, if it causes style regressions, others without context can track down the root cause. This would go under `bug fixes` in the `unreleased` doc.

Reply:
👍